clients/sample-app/src/components/Home/index.tsx

```diff
@@ -69,9 +69,9 @@ const Home = ({ privacyCenterUrl, products }: Props) => {
           </a>
         </div>
         <div>
-          <a className={css.modalLink} href="#" id="fides-modal-link">
-            Manage preferences
-          </a>
+          <button type="button" className={css.modalLink} id="fides-modal-link">
```

Comment:
❤️ thanks for making it a button. You can remove the eslint-disable rule on line 1 in this file now, I added that to silence the linter!